src/fixture.js

```diff
@@ -48,16 +48,24 @@ function specFromFixture(description, inputs) {
       }
 
       var chart = _acquireChart(config, json.options);
-      if (!inputs.png) {
-        fail(descr + '\r\nMissing PNG comparison file for ' + input);
-        done();
-      }
+      const _done = () => {
+        if (!inputs.png) {
+          fail(descr + '\r\nMissing PNG comparison file for ' + input);
+          done();
+        }
 
-      readImageData(inputs.png, function(expected) {
-        expect(chart).toEqualImageData(expected, json);
-        _releaseChart(chart);
-        done();
-      });
+        readImageData(inputs.png, function(expected) {
+          expect(chart).toEqualImageData(expected, json);
+          _releaseChart(chart);
+          done();
+        });
+      };
+      const run = json.options && json.options.run;
+      if (typeof run === 'function') {
+        json.options.run(chart, _done);
```

Comment:
Could use a Promise instead ;)

```js
Promise.resolve(json.options.run(chart)).finally(_done);
```

Reply:
True. Could make the `triggerMouseEvent` async too, while at it?

Reply:
And lets leave that to next version, so we have a working version for datalabels-master.